docs/docs/core/permissions/frontend/fetching-permissions.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Fetching Permissions
+tags:
+  - permissions
+  - RBAC
+---
+
+This document lays out the code-flow of fetching the permissions in the CMS and how the data is then injected in to the app
+to be used by components.
+
+## Where do we fetch them?
+
+At the very root of the entire admin panel we handle 4 routes:
+
+- Signing into the app
+- Not Found
+- Collecting information from a first time user
+- The actual app
```

Comment:
I think I would be more specific about the actual app being the Authenticated app, wdyt? 👀